recipes/httpx-negotiate-sspi/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "httpx-negotiate-sspi" %}
+{% set version = "0.26.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/httpx_negotiate_sspi-{{ version }}.tar.gz
+  sha256: cbf5ae118f33c92986eedffe18913e46252788ae97cd43f75f8efbad5725d0f4
+
+build:
+  skip: True  # [not win]
```

Comment:
We can make this an OS-noarch. See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-packages-with-os-specific-dependencies

Reply:
Ah, cool. Thanks will make it `noarch`...

Reply:
Actually @ocefpaf, this package only makes sense to install on Windows - it uses Windows specific apis that are not available on Linux.

IIUC the Linux equivalent would be `GSSAPI` which can work with both platforms but requires a workign Kerberos configuration. Windows users tend to just use SSPI because if works out of the box on that platform.

I'm publishing both packages here so that on Windows users can use `httpx-negotiate-sspi` and on Linux (where that doesn't work) they can instead fallback to `httpx-ntlm`.

Given that it doesn't work on Linux should it still be `noarch`?

Reply:
> Given that it doesn't work on Linux should it still be `noarch`?

If it makes the Linux package effectively a dummy package that might be a good thing as it then won't force all packages which depend on it to also be arch-specific 🤔 

The import would fail, but likely for a `ModuleNotFound` for `pywin32` rather than `httpx_negotiate_sspi`:
```python
from httpx_negotiate_sspi import HttpNegotiateAuth
```

Reply:
You can make it windows only. The reason for os-niarch is to avoid the Python versions matrix.